master/buildbot/www/service.py

```diff
@@ -132,6 +133,12 @@ def uid(self):
         return jwt.encode(claims, self.site.session_secret, algorithm=SESSION_SECRET_ALGORITHM)
 
 
+class BuildbotLogFile(LogFile):
```

Comment:
Is this really needed?

I think this is rather an issue with twisted's LogFile.